### PR TITLE
Resilience to no comments

### PR DESCRIPTION
**Pull request by luizmb**

### CODE REVIEW

File: Tests/MergeL10nTests/ParserTests.swift [hunk 1]
```diff
@@ -96,4 +96,116 @@ class ParserTests: XCTestCase {
...
+
+        let expectedResult = [
+            LocalizedStringEntry(
+                key: "L10n.MyModule.SomeView.title",
+                value: "This is the title",
+                comment: "Comment for title")
+        ]
+
+        let (match, rest) = LocalizedStringFile.parser().run(file)
+
```
Comment (zweigraf) [inline Tests/MergeL10nTests/ParserTests.swift:120]: Can you put „requiresComments: true“ manually here? I could not match test name + implementation at first. Would make it clearer, I think.

**Review by melle — APPROVED** [review]

**Review by zweigraf — APPROVED** [review]
🚀

File: Tests/MergeL10nTests/ParserTests.swift (hunk 1 above)
Comment (luizmb) [reply Tests/MergeL10nTests/ParserTests.swift:120]: Good point. One sec.

Comment (luizmb) [reply Tests/MergeL10nTests/ParserTests.swift:120]: done!

**Pull request merged**